index.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <img src="/images/logo.svg" alt="Logo" />
+      <img src="/images/menu-icon.svg" alt="Menu Icon" />
+    </header>
+    <main class="hero-section">
+      <section>
+        <h1 class="hero-heading">I'm Zain Glad to see you!</h1>
+        <p class="hero-paragraph">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h3 class="hero-caption">Let's Connect</h3>
+        <div>
+          <ul class="hero-social-media">
+            <li>
+              <img src="/images/Twitter.svg" alt="Twitter" />
+            </li>
+            <li>
+              <img src="/images/LinkedIn-Icon.svg" alt="LinkedIn" />
+            </li>
+            <li>
+              <img src="/images/Medium.svg" alt="Medium" />
+            </li>
+            <li>
+              <img src="/images/Github.svg" alt="Github" />
+            </li>
+            <li>
+              <img src="/images/Victory.svg" alt="Victory" />
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+
+    <section class="grid-container">
+      <div class="work-card">
```

Comment:
- [ ] Your list of projects grouped using `div` tags. Kindly use `ul` tag to group them, and `li` per project. This will enhance a cleaner syntax

Reply:
Alright man